integration-tests/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
There is a top-level .gitignore file, so this should be redundant

Reply:
deleted this file